gnd/src/main/java/com/google/android/gnd/ui/home/mapcontainer/MapContainerViewModel.java

```diff
@@ -96,6 +96,10 @@
   @Hot(replays = true)
   private final MutableLiveData<Integer> mapControlsVisibility = new MutableLiveData<>(VISIBLE);
 
+  private final MutableLiveData<Boolean> polygonDrawingCompleted = new MutableLiveData<>(false);
```

Comment:
Since the view model literally models the view,  this should be named specifically for what it represents in the UI: "completeButtonVisible".

Also, please use "visible" here instead of "visibility" to avoid confusion with the int view visibility.

Reply:
updated